packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -50,6 +56,15 @@ export default class NextjsInitializer implements Initializer {
       addInitializers = addInitAnswer.addInitializers;
     }
 
+    if (!addInitializers.includes('nextjs-styleguide')) {
+      const pkgPath = path.resolve(`${answers.destination}${sep}package.json`);
+      const pkg = openPackageJson(pkgPath);
+
+      pkg.scripts.bootstrap = pkg.scripts.bootstrap.replace(' && graphql-let', '');
+
+      writeFileToPath(pkgPath, JSON.stringify(pkg, null, 2));
```

Comment:
We have `writePackageJson` helper also. Should we use that instead here?

Reply:
Sure, I will